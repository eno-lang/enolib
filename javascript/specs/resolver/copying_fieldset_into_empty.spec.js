const { parse } = require('../..');

const input = `
languages:
eno = eno notation
json = json object notation

copy < languages
`.trim();

describe('Resolution', () => {
  describe('Copying fieldset into empty', () => {
    it('works', () => {
      const doc = parse(input);
      expect(doc.raw()).toMatchSnapshot();
    });
  });
});
