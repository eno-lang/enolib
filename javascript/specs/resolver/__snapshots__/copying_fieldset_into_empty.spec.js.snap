// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Resolution Copying fieldset into empty works 1`] = `
Object {
  "elements": Array [
    Object {
      "entries": Array [
        Object {
          "key": "eno",
          "type": "fieldsetEntry",
          "value": "eno notation",
        },
        Object {
          "key": "json",
          "type": "fieldsetEntry",
          "value": "json object notation",
        },
      ],
      "key": "languages",
      "type": "fieldset",
    },
    Object {
      "entries": Array [
        Object {
          "key": "eno",
          "type": "fieldsetEntry",
          "value": "eno notation",
        },
        Object {
          "key": "json",
          "type": "fieldsetEntry",
          "value": "json object notation",
        },
      ],
      "key": "copy",
      "type": "fieldset",
    },
  ],
  "type": "document",
}
`;
